package.json

```diff
@@ -1,40 +1,57 @@
 {
   "name": "ember-introjs",
-  "version": "1.0.0",
-  "description": "The default blueprint for ember-cli addons.",
+  "version": "2.0.0",
+  "description": "Intro.js wrapper component",
```

Comment:
In GitHub description, we have: `An Ember Component for intro.js` Shouldn't we keep that unified?